lib/vanagon/engine/base.rb

```diff
@@ -10,7 +10,7 @@ def initialize(platform, target = nil)
         @platform = platform
         @required_attributes = ["ssh_port"]
         @target = target if target
-        @target_user = "root"
+        @target_user = @platform.target_user
```

Comment:
@haus it seems like this is the line causing issues. It seems like this is calling the method and for some reason passing it in to the vmpooler token check, which is really confusing me. Everything else works just fine, just not tests

```
  1) Vanagon::Engine::Pooler #load_token prefers an env var to a file
     Failure/Error: expect(Vanagon::Engine::Pooler.new(platform).token).to eq('abcd')
       #<Double Vanagon::Platform> received unexpected message :target_user with (no args)
     # ./lib/vanagon/engine/base.rb:13:in `initialize'
     # ./lib/vanagon/engine/pooler.rb:12:in `initialize'
     # ./spec/lib/vanagon/engine/pooler_spec.rb:29:in `new'
     # ./spec/lib/vanagon/engine/pooler_spec.rb:29:in `block (3 levels) in <top (required)>'
```